cypress.config.js

```diff
@@ -7,7 +7,7 @@ module.exports = defineConfig({
     baseUrl: 'http://localhost:3000',
     viewportHeight: 800,
     viewportWidth: 1800,
-    numTestsKeptInMemory: 0,
+    numTestsKeptInMemory: 5,
```

Comment:
Why was this changed?

Reply:
Sorry, I changed this while debugging and forgot to set it back to 0 when pushing. We can put it back to 0.